src/components/commons/Footer.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+<footer class="footer footer-center p-10 text-primary-content ossph-gradient-bg">
+  <div class="grid grid-flow-row sm:grid-flow-col gap-4">
+    <a href="https://discord.com/servers/open-source-software-ph-905496362982981723"
+      target="_blank">
+      <button class="btn btn-ghost text-white">
```

Comment:
Please set button width to full/block on mobile or small screens.

![image](https://github.com/OSSPhilippines/ossph.org/assets/8638243/3946914f-86d1-4db2-898e-f4791d607760)

Reply:
cge boss. good idea

Reply:
Applied this behavior

**Wide screen**
![image](https://github.com/OSSPhilippines/ossph.org/assets/1091401/c1b9c454-c384-463e-bdaf-5bcb592a8ce0)

**Smaller screens**
![image](https://github.com/OSSPhilippines/ossph.org/assets/1091401/e121643b-3da5-4b5e-8456-b0aefe5fd38a)